dallinger/recruiters.py

```diff
@@ -171,8 +200,19 @@ def close_recruitment(self):
         """
         logger.info("Close recruitment.")
 
+    def _extract_blacklist_from_config(self):
+        # At some point we'll support lists, so all service code supports them,
+        # but the config system only supports strings for now, so we convert:
+        blacklist = self.config.get('blacklist')
+        if blacklist:
+            blacklist = tuple([item.strip() for item in blacklist.split(',')])
+        else:
+            blacklist = ()
+
+        return blacklist
```

Comment:
@suchow and I briefly debated whether to change the config system at this point, and the quick decision was to defer that, but if this story seems to be working otherwise, this week may be the time to do it.